JOSESwift/Sources/JWK.swift

```diff
@@ -53,15 +53,15 @@ public protocol JWK: Codable {
     /// The parameters of the JWK representing the properties of the key(s), including the value(s).
     /// Check [RFC 7517, Section 4](https://tools.ietf.org/html/rfc7517#section-4) and
     /// [RFC 7518, Section 6](https://tools.ietf.org/html/rfc7518#section-6) for possible parameters.
-    var parameters: [String: String] { get }
+    var parameters: [String: JWKParameterType] { get }
 
     /// Accesses the specified parameter.
     /// The parameters of the JWK representing the properties of the key(s), including the value(s).
     /// Check [RFC 7517, Section 4](https://tools.ietf.org/html/rfc7517#section-4) and
     /// [RFC 7518, Section 6](https://tools.ietf.org/html/rfc7518#section-6) for possible parameters.
     ///
     /// - Parameter parameter: The desired parameter.
-    subscript(parameter: String) -> String? { get }
+    subscript(parameter: String) -> Any? { get }
```

Comment:
Why use `Any?` here instead of `JWKParameterType?`? Are there non `[String]?` or `String?` return types?

Reply:
I think you're right! I'll need to investigate what I did exactly back then but yeah I think you're right. Will update.

The whole parameter stuff in JWK and in the JOSE header is not exactly nice. We'd love to update all of that some time. 🙏